IQKeyboardManager/IQKeyboardManager.m

```diff
@@ -1127,17 +1127,9 @@ - (void)keyboardDidShow:(NSNotification*)aNotification
     UIViewController *controller = [_textFieldView topMostController];
     if (controller == nil)  controller = [[self keyWindow] topMostWindowController];
 
-    //If _textFieldView viewController is presented as formSheet, then adjustFrame again because iOS internally update formSheet frame on keyboardShown. (Bug ID: #37, #74, #76)
```

Comment:
@ChristianPraiss Could you please explain why you removed these validations?

Reply:
The fix would have been adding the same code in the else branch, so we don't need the validation anymore, as both branches would execute the same code.

Reply:
In which else branch? Can you point out?

Reply:
Before the fix we checked if the viewController was presented modally and then called adjustFrame, but since the fix was calling adjustFrame when the viewController was not presented modally, too, we can get rid of that check